packages/cactus-plugin-ledger-connector-polkadot/src/test/typescript/integration/api-surface.test.ts

```diff
@@ -0,0 +1,8 @@
+import test, { Test } from "tape-promise/tape";
```

Comment:
@RafaelAPB Any chance that you could write the tests with Jest? See my earlier email on the mailing list for rationale (we've migrated over last summer)

Reply:
Please see comment bellow